scripts/dev_setup.sh

```diff
@@ -589,6 +593,24 @@ function install_deno {
   chmod +x "${INSTALL_DIR}deno"
 }
 
+function install_swift {
+    echo Installing Swift.
+    #if [[ "$PACKAGE_MANAGER" == "apt-get" ]]; then
+        #apt-get install binutils git gnupg2 libc6-dev libcurl4 libedit2 libgcc-9-dev libpython2.7 libsqlite3-0 libstdc++-9-dev libxml2 libz3-dev pkg-config tzdata zlib1g-dev -y
+        #wget https://swift.org/builds/swift-5.3.3-release/ubuntu2004/swift-5.3.3-RELEASE/swift-5.3.3-RELEASE-ubuntu20.04.tar.gz
```

Comment:
intend to leave a comment here?

Reply:
Currently working on adding CI support in this PR (and testing with the CI install process), so the dev-setup.sh script might be a bit messy while I'm debugging things with it...